_test/common_functions/simulate_spe_testfunc.m

```diff
@@ -32,7 +32,11 @@
 [~,seed_key] = fileparts(spe_file);
 sk = regexp(seed_key,'(\d+)','tokens');
 sk = [sk{:}];
-seed_key =['key_', [sk{:}]];
+if isempty(sk)

```

Comment:
How does this random number seed management work alongside the Herbert `_test\shared\seed_rng` (available to Herbert and Horace tests) @hsautessella ?

Reply:
Doesn't look like this seed ever touches Matlab's `rand` functions. It's only used in `rand_like` (defined in `Herbert/herbert_core/utilities/maths`).

Also, `seed_rng` lives in `test_tobyfit` at the moment. We need to sort out an `init_tests` setting in Horace, so we can add the right directories to the Matlab path when we're running Horace tests

Reply:
I think these two are completely independent. I am using one, deployed in gen_sqw